Makefile.config.in

```diff
@@ -182,6 +182,9 @@ INSTALL_SOURCE_ARTIFACTS=@install_source_artifacts@
 OC_CFLAGS=@oc_cflags@
 CFLAGS?=@CFLAGS@
 OC_CPPFLAGS=-I$(ROOTDIR)/runtime @oc_cppflags@
+OC_NATIVE_CPPFLAGS=\
+  -DNATIVE_CODE -DTARGET_$(ARCH) -DMODEL_$(MODEL) -DSYS_$(SYSTEM)
```

Comment:
Slight bike-shedding of the name... `OC_NATIVE_CPPFLAGS` sounds like an alternative to `OC_CPPFLAGS` when in fact these are added to it. Perhaps `OC_ADDITIONAL_NATIVE_CPPFLAGS`?